python/lsst/ts/idl/enums/MTMount.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of ts_idl.
+#
+# Developed for the LSST Data Management System.
```

Comment:
Not sure if ts_idl should refer to Vera Rubin Observatory as well instead of LSST but I'd recommend to at least change it in this file.

Reply:
Good suggestion. I changed all of the python files.